docs/release-testing.md

```diff
@@ -132,11 +125,11 @@ $ docker build -f Dockerfile . -t quay.io/$USERNAME/koku-metrics-operator-bundle
 6. Create an opm index (test-catalog) that contains the last release and the new release, and push to Quay:
 
 ```sh
-$ opm index add --bundles quay.io/$USERNAME/koku-metrics-operator-bundle:v$PREVIOUS_VERSION,quay.io/$USERNAME/koku-metrics-operator-bundle:v$VERSION --tag quay.io/$USERNAME/test-catalog:latest --container-tool docker
-$ docker push quay.io/$USERNAME/test-catalog:latest
+$ opm index add --from-index quay.io/project-koku/kmc-test-catalog:v$PREVIOUS_VERSION --bundles quay.io/$USERNAME/koku-metrics-operator-bundle:v$VERSION --tag quay.io/$USERNAME/test-catalog:latest --container-tool docker
+$ docker push quay.io/$USERNAME/test-catalog:v$VERSION
 ```
 
-7. Check that the test-catalog in OCP contains the new version.
+7. Update the image in the CatalogSource to point to the new test-catalog version (i.e. `image: quay.io/$USERNAME/test-catalog:v$VERSION`).
```

Comment:
Since the other testing instructions use `quay.io/project-koku/kmc-test-catalog:v$VERSION` should we also use that here?

Reply:
I did this so that the new catalog we are testing with comes from a personal repo. I think we should keep `kmc-test-catalog` only for catalogs that we know are good and represent what is currently in the community-operator catalog.

I think after we have verified that the new catalog is good, then we can add the new version to `kmc-test-catalog`.

Reply:
That makes sense to me!